about.html

```diff
@@ -0,0 +1,118 @@
+                        </a>
+                    </div>
+                </div>
+                <div class="menu-button-mobile"></div>
+            </nav>
+            <div id="title-bar">
+                <h1 class="title">Applicatie</h1>
+                <div class="breadcrumb"><a href="index.html" title="home">Home</a>-&gt;<a href="about.html" title="Over ons">Over ons</a></div>
+            </div>
+            <div id="content">
+                <div id="video">Video plaatsen</div>
+                <h1>Over Bewust Kopen</h1>
+                <p>                         
+                    Men is vandaag de dag steeds meer bezig met bewuste keuzes maken als het gaat om voeding.
+                    Hoe kun je er zeker van zijn dat wanneer je een product koopt, er ook niets in zit wat je niet wil of mag eten. Wanneer eet je iets wat nu echt vegetarisch is? Welke producten voldoen nu 100% aan het kenmerk vegetarisch? Vaak wordt er bij producten vanuit gegaan omdat hier geen vleesproducten of bijvoorbeeld bouillon in zit dat deze ook vegetarisch zijn. Dit terwijl er in drop gelatine verwerkt zit, wat gemaakt wordt van botten van dieren, dus niet 100% vegetarisch. Zo ook het drinken voor kinderen waar geen suiker, maar wel glucose aan toegevoegd is en dus niet zo gezond is als gepretendeerd wordt.
+                    Door het grote aanbod en onduidelijkheid over de herkomst van producten en ingrediënten is er een kennisachterstand ontstaan.
+                    Bewust kopen vind het belangrijk dat het eenvoudig wordt om echt te weten wat er nu in de producten zit. Of deze voldoen aan je eetpatroon en of deze eigenlijk wel voldoen aan wat het
+                    label vermeld. Wij willen duidelijkheid bieden over waar die e-nummers op het etiket nu voor staan en vandaan komen. Of als er suikervrij op de verpakking staat, maar er wel fructose in het product zit deze wel
+                    zo gezond is als gepresenteerd. Op deze manier willen we mensen kennis meegeven over de producten die ze kopen.Bewust Kopen wil assisteren om de keuze die u maakt echt een bewuste keuze te laten zijn.
+                </p>
+                <br><h3>Het team</h3>
+                <p>Bewust kopen bestaat op dit moment uit 3 jonge ondernemers:</p> 
+                
+                <div id="team"><ul>
+                        <li id="koen"><img src="images/koen.jpg" alt="Foto Koen" width="200"> 
+                            <p><h4>Koen Kukken</h4></p> </li>
+                        <li id="stephanie"><div><img src="images/stephanie.jpg" alt="Foto stephanie" width="200"> 
+                                <p><h4>Stephanie van der Heijden</h4></p></div> </li>
+                        <li id="freek"><img src="images/freek.jpg" alt="Foto Koen" width="200"> 
+                                <p><h4>Freek de Groen</h4></p> </li></ul>
+ 
+            
+            <footer>
```

Comment:
De footer wordt niet goed weergegeven. Misschien ergens een div niet goed genest of afgesloten?